lightning/src/ln/channel.rs

```diff
@@ -1119,6 +1119,81 @@ pub(crate) struct ShutdownResult {
 	pub(crate) channel_funding_txo: Option<OutPoint>,
 }
 
+#[derive(Debug, Copy, Clone)]
+enum HolderCommitmentPoint {
+	Uninitialized { transaction_number: u64 },
```

Comment:
#3109 should hopefully provide more context for this now

Reply:
Oh woops thought i responded to this - we basically preload pubkeys upon the creation of a node for all channels, so what we use for that probably can't directly help with the first commitment point. We could make it so that for async signing you need to pass the initial per commitment point into `ChannelManager::create_channel` or `accept_inbound_channel` to avoid doing the `Uninitialized` variant - and on our side we'd just have to wait for the response there which isn't a ton different from how we're waiting already